packages/core/src/components/axes/ruler.ts

```diff
@@ -24,10 +24,17 @@ export class Ruler extends Component {
 		domainValue: number;
 		originalData: any;
 	}[];
+	isRulerEnabled =  Tools.getProperty(
+		this.model.getOptions(),
+		"ruler",
+		"enabled"
+	);
 
 	render() {
 		this.drawBackdrop();
```

Comment:
it seems like `drawBackdrop` is repeated in ruler & grid. Would ruler fully function without it?

Reply:
ruler needs backdrop since the event listener is applied on backdrop.

Reply:
I get that ruler is dependant on backdrop, however I'm wondering if these lines are needed inside `ruler.ts`

![image](https://user-images.githubusercontent.com/14989804/91735248-04f62880-eb7a-11ea-879b-c864763d01cc.png)

Reply:
Actually, I am not sure about it. These lines of code have existed even before I added the ruler option.

Reply:
Yes I'm just asking whether removing them on your end still makes everything work?

Reply:
As what I can see do far, removing code within line 253 to line 261 won't break any thing on my end.

Reply:
Codes within line 253 to line 261 have already been removed.